settings.gradle

```diff
@@ -25,5 +25,3 @@ include ':retrofit-converters:simplexml'
 include ':retrofit-converters:wire'
 
 include ':samples'
-
-enableFeaturePreview("VERSION_CATALOGS")
```

Comment:
The [Version catalogs](https://docs.gradle.org/7.4/userguide/platforms.html) is promoted to a stable feature.